loadtest/failure_basic_invalid.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
can we juts use the same config json for different tests? the only diff seems to be `message_type`